public/css/app.css

```diff
@@ -660,49 +662,128 @@ form span {
   background-color: rgba(0, 0, 0, 0.38);
 }
 
-#false-door {
+body.show-subscribe-modal #subscribe-modal {
   opacity: 1;
   visibility: visible;
   pointer-events: all;
   cursor: pointer;
   transition: all 0.25s ease-out;
 }
 
-#false-door div {
+#subscribe-modal .modal-wrapper {
   margin: auto;
   border-radius: 2px;
   background: #fff;
   color: #000;
   display: inline-block;
+  padding: calc(30px + 1vw);
   width: 40vw;
-  min-width: 350px;
+  min-width: 300px;
   max-width: 600px;
-  padding: calc(3.33vw + 0.625rem) !important;
-  text-align: left;
+  text-align: center;
   box-sizing: border-box;
   box-shadow: 0 0 9px rgba(0, 0, 0, 0.18);
+  display: flex;
```

Comment:
Per **stylelint** this is a bit confusing, since we already had `display` defined above (see L678), with a value of "inline-block":

```
686:3   ✖  Unexpected duplicate "display"                                   declaration-block-no-duplicate-properties
```

Reply:
Probably what is going here is it needs an "alt" tag....